packages/marketplace/src/components/ui/app-revision-comparision/app-revision-comparision.tsx

```diff
@@ -8,10 +8,11 @@ import { DesktopIntegrationTypeModel, PagedResultDesktopIntegrationTypeModel_ }
 import DiffCheckbox from '../diff-checkbox'
 import DiffViewer from '../diff-viewer'
 import DiffRenderHTML from '../diff-render-html'
+import { DeveloperAppDetailState } from '@/reducers/developer'
 
 export type AppRevisionComparisionProps = {
   revisionDetailState: RevisionDetailState
-  appDetailState: AppDetailState
+  appDetailState: AppDetailState | DeveloperAppDetailState
```

Comment:
Avoid modifying this file as it's currently being used for our old AppDetails modal which will be deleted after the new standalone page is finished. Instead, create a new file in developer folder and start working on it.

Reply:
Small changes. Current it's modified to receive another different shape, no behaviour changes. Behaviour is the same. IMO, It should be refactored/modified when changes/behaviours addressed is big enough.